server/reportConfiguration/types/ATTEMPTED_ESCAPE_FROM_CUSTODY.ts

```diff
@@ -171,7 +171,7 @@ const ATTEMPTED_ESCAPE_FROM_CUSTODY: IncidentTypeConfiguration = {
           id: '179034',
           code: 'RULE 43 UNIT (GOAD)',
           active: true,
-          label: 'RULE 43 UNIT (GOAD)',
+          label: 'Rule 43 unit (goad)',
```

Comment:
google says GOAD might is another acronym (= good order and discipline)

Reply:
I saw this a few times but I didn't find anything useful. I'll add it as a preserved word as `GOAD` then unless you have objections.

Reply:
Updated here: https://github.com/ministryofjustice/hmpps-incident-reporting/pull/287/commits/790cdaf0d52e8865428c4d05bd127067f162acdf